base/sparse/sparsevector.jl

```diff
@@ -1679,3 +1679,43 @@ function sort{Tv,Ti}(x::SparseVector{Tv,Ti}; kws...)
     newnzvals = allvals[deleteat!(sinds[1:k],z)]
     SparseVector(n,newnzind,newnzvals)
 end
+
+function fkeep!{Tv,Ti}(x::SparseVector{Tv,Ti}, f, other, trim::Bool = true)
```

Comment:
Yes, kept the signature same as for matrices but it could be removed for both matrices and vectors.